### PR TITLE
Client - docker, js, html, css

### PR DESCRIPTION
**Pull request by Nicolaj2903**
Currently, the dockerfile installs Node.js, and deploys the application at port 3000

### CODE REVIEW

File: src/Client/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Application</title>
+    <style>
+        /* Optional: Add some basic styling */
```
Comment (Crazi12345) [inline src/Client/index.html]:
```suggestion
```

File: src/Client/index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        body {
+            font-family: Arial, sans-serif;
```
Comment (Crazi12345) [inline src/Client/index.html:11]: bad font

**Review by Crazi12345 — APPROVED** [review]
Approved

File: src/Client/index.html (hunk 2 above)
Comment (Nicolaj2903) [reply src/Client/index.html:11]: Suggestions for a better one? :)

**Pull request merged**